lib/liquid/standardfilters.rb

```diff
@@ -151,6 +151,20 @@ def sort_natural(input, property = nil)
       end
     end
 
+    # Filter the elements of an array to those with a certain property value.
+    # by default the target value is assumed to be `true`.
```

Comment:
Nit: capitalize "by" since it's the first word of a sentence. Remove "assumed to be". Change "true" to "truthy".